util/submitter_amqp.go

```diff
@@ -133,13 +134,9 @@ func MakeAMQPSubmitterWithReconnector(url string, target string, verbose bool,
 		if err != nil {
 			return nil, err
 		}
-		err = mySubmitter.connect()
-		if err != nil {
-			return nil, err
-		}
 
-		mySubmitter.Conn.NotifyClose(mySubmitter.ErrorChan)
 		go reconnectOnFailure(mySubmitter)
+		mySubmitter.ErrorChan <- utils.NewError(0, "Initial connect", true, true)
```

Comment:
// Need to submit an initial error to land in the reconnect loop

Reply:
Comment added, thanks for the hint